common/account_utils/src/validator_definitions.rs

```diff
@@ -3,9 +3,10 @@
 //! Serves as the source-of-truth of which validators this validator client should attempt (or not
 //! attempt) to load into the `crate::intialized_validators::InitializedValidators` struct.
 
-use crate::{create_with_600_perms, default_keystore_password_path, ZeroizeString};
+use crate::{default_keystore_password_path, ZeroizeString};
 use directory::ensure_dir_exists;
 use eth2_keystore::Keystore;
+use filesystem::{create_with_600_perms, Error as fsError};
```

Comment:
Generally identifiers starting with lowercase letters are reserved for values rather than types

```suggestion
use filesystem::{create_with_600_perms, Error as FsError};
```

Alternatively because it's only used in 1 place we could just write `filesystem::Error`

Reply:
done!